pom.xml

```diff
@@ -67,7 +67,7 @@
         <dependency>
             <groupId>org.technologybrewery.habushu</groupId>
             <artifactId>habushu-maven-plugin</artifactId>
-            <version>2.10.1-SNAPSHOT</version>
+            <version>2.12.1</version>
```

Comment:
@d-ryan-ashcraft should this point to a snapshot?

Reply:
We should use the latest released version